lib/flowLSP/FlowLanguageClient/createMiddleware.js

```diff
@@ -208,8 +208,11 @@ function formatHoverContent(value: ?vscode.Hover): ?vscode.Hover {
   // format content
   value.contents = value.contents.map((content) => {
     // NOTE: in our case, content is always MarkdownString
```

Comment:
This comment needs rephrasing, isn't it?

Reply:
@thymikee updated - thank you!